Assets/MixedRealityToolkit.Examples/Experimental/Demos/UX/BoundsControl/Scripts/BoundsControlRuntimeExample.cs

```diff
@@ -217,8 +217,8 @@ private IEnumerator Sequence()
                     childTransform.transform.localScale = new Vector3(baseScale * Random.Range(.5f, 2f), baseScale * Random.Range(.5f, 2f), baseScale * Random.Range(.5f, 2f));
                 }
 
-                bbox.LinksConfiguration.WireframeEdgeRadius = 1f;
-                bbox.CreateRig();
+                boundsControl.LinksConfiguration.WireframeEdgeRadius = 1f;
+                boundsControl.CreateRig();
```

Comment:
I can't leave a comment on the existing code here, but consider having SetStatus take an option boolean (i.e. bool hasNext) which control whether or not "Press '1' or say 'select' to continue" is actually shown.

I don't think that pressing 1 or saying select once you are on that state actually does anything, so I was a little confused until I looked at the code (i.e. otherwise, is it a bug? or is it expected?)

That way the last SetStatus:

Can just be:

```csharp
SetStatus("Done!", false /*showNextGuidance*/);
```